### ISSUE TITLE
Make Focus-visible on the carousel

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5278]
from a11y audit #5152

Manual testing found that the carousel focus is not visible or has poor contrast on all three themes.  This was addressed for the clean theme in #4872, (I have tested that at time of writing, and it is different to the situation at the a11y-audit-1.9 tag., so a bug has been introduced .  A quick fix for clean could be to reintroduce changes from that work.

**Comment by ajrbyers** [comment]
Closed by https://github.com/openlibhums/janeway/pull/5521
